test/ng/snifferSpec.js

```diff
@@ -25,6 +25,25 @@ describe('$sniffer', function() {
     });
 
 
+    it('should be true if running inside nw.js', function() {
```

Comment:
I put it there to make sure both tests verifying when history should be true are together. But I can move it a bit lower. I wouldn't incorporate it into the same test that verifies history being false.